{"uuid": "fae691d3-a702-4ea5-8a0d-9fb8b676ce4e", "children": ["c493b982-77e4-4ac2-b2f9-823ff45f8c31"], "befores": [{"name": "reg_user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 500 == 201\n +  where 500 = <Response [500]>.status_code\n", "trace": "  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nehaarchy/MyAPIFramework/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/nehaarchy/MyAPIFramework/API_Tests/test_setupteardown.py\", line 21, in reg_user\n    assert reg_response.status_code==201\n"}, "start": 1753297834280, "stop": 1753297841224}], "start": 1753297834280, "stop": 1753297841242}